src/js/providers/tracks-mixin.js

```diff
@@ -123,16 +123,19 @@ define(['utils/underscore',
             }
         }
 
-        if (this.renderNatively) {
-            // Only bind and set this.textTrackChangeHandler once so that removeEventListener works
-            this.textTrackChangeHandler = this.textTrackChangeHandler || textTrackChangeHandler.bind(this);
-            this.addTracksListener(this.video.textTracks, 'change', this.textTrackChangeHandler);
-
-            if (utils.isEdge() || utils.isFF() || utils.isSafari()) {
-                // Listen for TextTracks added to the videotag after the onloadeddata event in Edge and Firefox
-                this.addTrackHandler = this.addTrackHandler || addTrackHandler.bind(this);
-                this.addTracksListener(this.video.textTracks, 'addtrack', this.addTrackHandler);
-            }
+        // Only bind and set this.textTrackChangeHandler once so that removeEventListener works
+        this.textTrackChangeHandler = this.textTrackChangeHandler || textTrackChangeHandler.bind(this);
+        this.addTracksListener(this.video.textTracks, 'change', this.textTrackChangeHandler);
+
+        if (!this.renderNatively || utils.isEdge() || utils.isIE(11) || utils.isFF() || utils.isSafari()) {
```

Comment:
You can call `addTextTracks` in the provider to add subtitle tracks in IE11. For 608 captions or ID3 metadata, use `addVTTCue`. The mixin will create a track if one doesn't exist.